### ISSUE TITLE
Quick search fine tuning

### ISSUE THREAD

**Issue by miohtama** [tradingstrategy-ai/search#3]
<img width="1042" alt="image" src="https://user-images.githubusercontent.com/49922/157500653-4fbe10d2-877b-4a48-8a77-3796d21d9cab.png">

- Let's change the pair results above token (fow now, we will switch it back later when we have better token data available)
- Pairs are in the wrong order - the bottom most MATIC pair should be the first (it has liquidity and actual trading action)

**Comment by kenkunz** [comment]
> Let's change the pair results above token (for now, we will switch it back later when we have better token data available)

The ordering by type is handled by the `type_rank` field in the search index (see: [trading-entities.md](https://github.com/tradingstrategy-ai/search/blob/main/docs/trading-entities.md)). Re-importing the collection with `pair=2` and `token=3` should fix this. You could also leave `pair=3` and just set `token=4` or higher.

**Comment by kenkunz** [comment]
> Pairs are in the wrong order - the bottom most MATIC pair should be the first (it has liquidity and actual trading action)

Good news – this is fixed already on the [Advanced Search PR](https://github.com/tradingstrategy-ai/frontend/pull/16) – see screenshot:

<img src="https://user-images.githubusercontent.com/35901/157533584-cc01deb9-3a41-4570-899e-dfd5855d588b.jpeg" alt="search - matic" width="350">

The `sort_by` param was fine, but I changed the `query_by` params as I experimented with options while working on advanced search.

**`sort_by`** on both branches:
```
type_rank:asc,_text_match:desc,volume_24h:desc
```

**`query_by`** on original search PR (live in Prod):
```
name,token_tickers,token_names,smart_contract_addresses
```

**`query_by`** on Advanced Search PR:
```
description,token_tickers,token_names,smart_contract_addresses
```

Small change to look at `description` instead of `name` field… but seems to yield better results. It also ensures that other data that's in the description (like `blockchain` when relevant) is searchable. We can tune this further if needed. It would be trivial to extract the default `sort_by` and `query_by` to `ENV` vars so they're configurable w/out a code change.

**Comment by kenkunz** [comment]
@miohtama I am closing this – 1st bullet was addressed by search index update; 2nd one will be addressed with [frontend #16](https://github.com/tradingstrategy-ai/frontend/pull/16)
